### DISCUSSION TITLE
Build for Linux Mint 20.x

### DISCUSSION THREAD

**Discussion by Perkolator** [rhash/RHash#242] [Q&A] [answered]
I was finally able to build RHash for my Linux Mint 20.3 Cinnamon and update RHash from old 1.3.9 offered by the package manager to the newest version. I tried to build RHash in the past but I had some problems and dropped the idea completely because I had so many other things to learn about linux. The [INSTALL.md](https://github.com/rhash/RHash/blob/master/INSTALL.md) is old and hard to decipher.

So, this is the way I was able to build & install RHash 1.4.4 (first I uninstalled the old RHash (`sudo apt autopurge rhash`)):

```
sudo apt install build-essential
sudo apt install libssl-dev
./configure --enable-static=librhash --enable-openssl=runtime && make
sudo make install
```

I didn't see any errors in the configure & make outputs, and I haven't seen any errors running the new RHash version.

Biggest problem was the "configure" part. **Does it look ok?**

**Comment by rhash** [comment, marked as answer]
Yes.
